src/generator/__snapshots__/errata.integration.js.snap

```diff
@@ -44,7 +44,7 @@ exports[`aggregates multiple errata into a single play 1`] = `
   become: true
   tasks:
     - name: check for update
-      shell: \\"{{ ansible_facts['pkg_mgr'] }} check-update -q --advisory {{ insights_issues | regex_search('RH[SBE]A-20[\\\\\\\\d]{2}:[\\\\\\\\d]{4,5}( --advisory RH[SBE]A-20[\\\\\\\\d]{2}:[\\\\\\\\d]{4,5})*') }}\\"
+      shell: \\"{{ ansible_facts['pkg_mgr'] }} check-update -q --advisory {{ insights_issues | regex_search('(--advisory (RH[SBE]A-20[\\\\\\\\d]{2}:[\\\\\\\\d]{4,5}))+') }}\\"
```

Comment:
@semtexzv With this current change this shell command would be running:
--advisory --advisory (REGEX FOR RHSBE)

Which would not work.  

Because of the changes done to the AggregatedErrataPlays file we should be able to remove the first --advisory from all of these changed templates